examples/Component/MechanicalLoad/ConstantForceField.scn

```diff
@@ -13,9 +13,10 @@
     <RequiredPlugin name="Sofa.GL.Component.Rendering3D"/> <!-- Needed to use components [OglModel] -->
     <!-- Constant force for a deformable -->
     <VisualStyle displayFlags="showBehaviorModels showForceFields" />
-    <DefaultAnimationLoop/>
+    <DefaultAnimationLoop computeBoundingBox="0"/>
```

Comment:
```suggestion
    <DefaultAnimationLoop computeBoundingBox="true"/>
```

Reply:
@alxbilger why putting the bbox to true?